specs/agents/mobile/README.md

```diff
@@ -20,56 +20,57 @@ However, the Elastic mobile agents don't set every attribute defined (many only
+| [`telemetry.sdk.language`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/#telemetry-sdk)| `service.language.name`   | `swift`, `java`              | :white_check_mark: yes | Elastic's `agent.name` is mapped from:  `telemetry.sdk.name`/`telemetry.sdk.language` |
+| [`os.description`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)       | `host.os.full`             | `iOS Version 15.5 (Build 19F70)`      | :x: no | |
+| [`os.type`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)             | `host.os.platform` | `darwin` | :white_check_mark: yes |  |
+| [`os.version`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)           | `host.os.version` | `15.5.0`        | :x: no | |
+| [`os.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)              | `host.os.name` and `host.os.type` | `iOS`, `Android`   | :white_check_mark: yes |  |
+| [`deployment.environment`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/deployment_environment/) | `service.environment`    | `production`, `dev`              | :x: no |    |
+| [`deivce.id`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)        | `device.id`        | `E733F41E-DF47-4BB4-AAF0-FD784FD95653` | :x: no |  [Follow this description.](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/) (Device ID should be fix and unique for a device but should not carry PII)  |
+| [`device.model.identifier`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/) | `device.model.identifier` | `iPhone4`,`SM-G920F`            | :x: no  |             |
+| [`device.model.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)       | `device.model.name` | `Samsung Galaxy S6`            | :x: no  | This information is potentially not directly available on the device and needs to be derived / mapped from `device.model.identifier`. In this case, the APM server should do the mapping. |
+| [`device.manufacturer`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)     | `device.manufacturer` | `Apple`, `Samsung`            | :x: no  |  This information is potentially not directly available on the device and needs to be derived / mapped from `device.model.identifier`. In this case, the APM server should do the mapping. |   
+| [`process.runtime.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/process/#process-runtimes) | `service.runtime.name` | `Android Runtime` | :x: no | Use `Android Runtime` (Dalvik) and corresponding Dalvik version for Android.  |
```

Comment:
Should we state here to use `System.getProperty("java.runtime.name")`? I just confirmed using Android Studio that it returns "Android Runtime", while `java.vm.name` is "Dalvik".

Reply:
I think we could even just hard code it in the Android agent, right? Since with the move from Dalvik to ART the versioning stayed the same / continued. So we could just hardcode `Android Runtime`

Reply:
Fair enough!